### PR TITLE
solution

### PR DESCRIPTION
**Pull request by Andrey1104**

### CODE REVIEW

File: app/managers.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connect = sqlite3.connect("cinema")
+        self.table = "actors"
+
+    @staticmethod
+    def check_name(first_name: str, last_name: str) -> list:
+        if isinstance(first_name and last_name, str):
```
Comment (Dimosphen1) [inline app/managers.py:13]: This check will not work correctly, separate this check between two conditions. E.g., `isinstance(3 and "a", str)` will return `True`.

**Review by Dimosphen1 — CHANGES_REQUESTED** [review]
Several changes were requested.

File: app/managers.py (hunk 1 above)
Comment (Andrey1104) [reply app/managers.py:13]: this is so sad, I thought it would work

File: app/managers.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+            f"INSERT INTO {self.table} "
+            "(first_name, last_name) "
+            "VALUES (?, ?)",
+            (*self.check_name(first_name, last_name),)
+        )
+        self._connect.commit()
+
+    def all(self) -> list:
+        cursor = self._connect.execute(
+            f"SELECT id, first_name, last_name FROM {self.table};"
```
Comment (pavlejviki) [inline app/managers.py:29]: you can use * to retrieve all columns from a table

**Review by pavlejviki — CHANGES_REQUESTED** [review]

**Review by Polyakiv-Andrey — APPROVED** [review]
Good job!

**Pull request closed without merging**